.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
     - stable
-
+script:
+  - node ./validation/markets-validator.js
```

Comment:
You need to change the `package.json` and include it in the `scripts` property as explained here: http://browsenpm.org/package.json#scripts.test